sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md

```diff
@@ -570,8 +574,11 @@ Accept wildcard characters: False
 ```
 
 ### -DisableCompanyWideSharingLinks
+Disables People in your organization links. For more information, see [People in your organization sharing links](https://docs.microsoft.com/en-us/microsoft-365/solutions/microsoft-365-limit-sharing?view=o365-worldwide#people-in-your-organization-sharing-links).
+Possible values
 
-PARAMVALUE: Unknown | Disabled | NotDisabled
+- Disabled
+- NotDisabled
```

Comment:
Unknow is available. Maybe we should leave it.

Reply:
yes, but it not "settable", what should we do? unknown is a state only when you have never set this.

Reply:
Good question... not sure if it's better to leave it and add a comment in brackets like "not settable" or what you propose. @tseward?

Reply:
I agree with a not settable type verbiage.